docker/package/packages.py

```diff
@@ -17,10 +17,14 @@
 networks = {
     "mainnet": "mainnet",
     "jakartanet": "jakartanet",
+    "ghostnet": "ghostnet",
+    "kathmandunet": "kathmandunet",
```

Comment:
Sadly, v14.0-rc1 `tezos-node` doesn't have predefined kathmandunet :confused: 
```suggestion
    "kathmandunet": "https://teztnets.xyz/kathmandunet",
```